packages/components/src/components/text/text.jsx

```diff
@@ -1,18 +1,18 @@
 import classNames from 'classnames';
 import React from 'react';
 
-const Text = ({ children, size, color, align, weight, lineHeight, as, className }) => {
+const Text = ({ children, size, color, align, weight, lineHeight, flex, as, className }) => {
```

Comment:
(rest-props) Please add `...props` and pass it down to the rendered element, as any additional property is not being passed down.